spyder/plugins/editor/widgets/editorstack/editorstack.py

```diff
@@ -343,6 +351,43 @@ def show_in_external_file_explorer(self, fnames=None):
                 QMessageBox.information(self, msg_title, msg,
                                         QMessageBox.Ok)
 
+    def copy_absolute_path(self):
+        """
+        Copy current filename absolute path to the clipboard.
+
+        Returns
+        -------
+        None.
+
+        """
+        QApplication.clipboard().setText(
+            self.get_current_filename())
+
+    def copy_relative_path(self):
+        """
+        Copy current filename relative path to the clipboard.
+
+        Returns
+        -------
+        None.
+
+        """
+        file_drive = osp.splitdrive(self.get_current_filename())[0]
+        if (os.name == 'nt'
+                and osp.splitdrive(getcwd_or_home())[0] != file_drive):
+            QMessageBox.warning(
+               self,
+               _("No available relative path"),
+               _("It is not possible to copy a relative path "
+                 "for this file because it is placed in a "
+                 "different drive than your current working "
+                 "directory. Please copy its absolute path.")
+            )
+        else:
+            QApplication.clipboard().setText(
+                (osp.relpath(self.get_current_filename(),
+                             getcwd_or_home()).replace(os.sep, "/")))
```

Comment:
```suggestion
            rel_path = osp.relpath(
                self.get_current_filename(), getcwd_or_home()
            ).replace(os.sep, "/")
            QApplication.clipboard().setText(rel_path)
```

Reply:
@dalthviz, this is still missing to get the file path relative to the project one in case there's an active project and the cwd is not the same as the project directory.